apps/web/scripts/kernel/utils/handler.ts

```diff
@@ -17,11 +18,11 @@ export const triggerActionToGetPrivateKey = async () => {
 		const response = await modules.storage.find(selectors.solanaKeys);
 		const [publicKey] = response.docs as PublicKeyDocument[];
 		const encryptedKey = await modules.storage.safeGet<PrivateKeyDocument>(
-			publicKey._id,
+			publicKey.privateKeyId,
 		);
 
 		return await decryptWithPasscode(
-			'123456',
+			'111111',
```

Comment:
smell something wrong here.

Reply:
this method is still being used in `suiHandler.ts`, would remove it after refactoring Sui handler